src/StackAdmin/Azs.AzureBridge.Admin/readme.md

```diff
@@ -57,16 +57,52 @@ csproj: Azs.AzureBridge.Admin.csproj
 psd1: Azs.AzureBridge.Admin.psd1
 psm1: Azs.AzureBridge.Admin.psm1
 
-directive:
+directive:  
+  # Add AzureBridge prefix to subjects to match existing method names
   - where:
-      model-name: ActivationResource
+      subject: Activation
     set:
-      suppress-format: true
+      subject-prefix: AzureBridge
+  - where:
+      subject: DownloadedProduct
+    set:
+      subject-prefix: AzureBridge
+  - where:
+      subject: Product
+    set:
+      subject-prefix: AzureBridge
+
+  # Add alias for ResourceGroup to ResourceGroupName
+  - where:
+      parameter-name: ResourceGroup
```

Comment:
i think we can change the swagger spec directly renaming resourceGroup to resourceGroupName

Reply:
Changes were made in azure-rest-api-specs repo